schemas/context/experienceevent-privacy.example.1.json

```diff
@@ -0,0 +1,11 @@
+{
+    "xdm:consentStrings": [
+        {
+            "xdm:consentStandard": "IAB TCF",
+            "xdm:consentStandardVersion": "2.0",
+            "xdm:consentStringValue": "BObdrPUOevsguAfDqFENCNAAAAAmeAAA.PVAfDObdrA.DqFENCAmeAENCDA",
```

Comment:
Having `consent` within `consentStandard`, `consentStandardVersion`, and `consentStringVersion` seems redundant since they are nested within `consentStrings`. It seems like `String` in `consentStringValue` is redundant is well. Would this be better?

```
{
  "xdm:consentStrings": [
    {
      "xdm:standard": "IAB TCF",
      "xdm:standardVersion": "2.0",
      "xdm:value": "BObdrPUOevsguAfDqFENCNAAAAAmeAAA.PVAfDObdrA.DqFENCAmeAENCDA",
      "xdm:gdprApplies": true,
      "xdm:gdprContainsPersonalData": false
    }
  ]
}
```

Reply:
No, that won't work because `xdm:standard`, etc. are too generic.